server/service/global_policies.go

```diff
@@ -164,13 +173,33 @@ func (svc Service) DeleteGlobalPolicies(ctx context.Context, ids []uint) ([]uint
 	if len(ids) == 0 {
 		return nil, nil
 	}
+	// Is there a better approach to gather policy names so we can include them in the activity records?
```

Comment:
Not at this level, but there is at the datastore level. You can create a function that does `select name from policy where id in (?)`.

Alternatively, you can add this functionality as part of what `ds.DeleteGlobalPolicies` returns.